packages/web-components/src/components/va-table/va-table.tsx

```diff
@@ -37,6 +55,30 @@ export class VaTable {
       item.setAttribute('scope', 'col');
     });
 
+    if (this.sortColumn >= 0) {
+      const icon = this.sortAscending ? ascendingIcon : descendingIcon;
+      const button = document.createElement('button');
+      button.setAttribute(
+        'aria-label',
+        `sort data by ${this.sortAscending ? 'descending' : 'ascending'}`,
+      );
+      button.classList.add(
+        'vads-u-padding--0',
+        'vads-u-margin--0',
+        'vads-u-text-decoration--none',
+        'vads-u-font-weight--bold',
+        'vads-u-background-color--gray-lightest',
+        'vads-u-color--base',
+      );
+      button.innerHTML = `${columns[this.sortColumn]}${icon}`;
+      button.onclick = this.handleSort.bind(this);
```

Comment:
I'm doing all of this stuff the old way instead of with JSX because this button will exist in the light DOM.